log2ram.conf

```diff
@@ -14,12 +14,12 @@ USE_RSYNC=false
 
 # If there are some errors with available RAM space, a system mail will be send
 # Change it to false and you will have only a log if there is no place on RAM anymore.
-MAIL=true
+MAIL=false
 
 # **************** Zram backing conf  *************************************************
 
 # ZL2R Zram Log 2 Ram enables a zram drive when ZL2R=true ZL2R=false is mem only tmpfs
-ZL2R=false
+ZL2R=true
```

Comment:
Don't change the default

Reply:
I have no idea why you are even commenting? I said give that a try?
IE give it a try not pull / push or anything.

I can change what I wish when I am demonstrating something especially the items which need to be seen for the demonstration.

Still haven't answered the important questions about writes or the start up that breaks applications expecting log files?